src/parser.ts

```diff
@@ -69,6 +69,37 @@ function parseNonLiteral(
 
   log(whiteBright.bgBlue('parser'), schema, '<-' + typeOfSchema(schema), processed.has(schema) ? '(FROM CACHE)' : '')
 
+  function parseArray(
+    standaloneName: string | undefined,
+    params: AST,
+    minItems: number | undefined,
+    maxItems: number | undefined
+  ): AST {
+    if ((typeof minItems === 'number' && minItems > 0) ||
```

Comment:
Is this something that you could move to the normalizer step, so that you unit test it in isolation?